Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,62 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
+        public static void Main()
+            {
+                PersonDetails();
+            }
+
+        public static void PersonDetails()
+            {
+            string firstName, lastName;
+            float age, weight, height;
+
+            Console.Write("Input First Name: ");
+            firstName = Console.ReadLine();
+            Console.Write("Input Last Name: ");
+            lastName = Console.ReadLine();
+
+            // use s and returnValue to tryparse strings to floats
+            string s = "";
+            float returnValue;
+
+            Console.Write("Input age: ");
+            s = Console.ReadLine();
+            float.TryParse(s, out returnValue);
+            age = returnValue;
+
+            Console.Write("Input weight (in kg): ");
+            s = Console.ReadLine();
+            float.TryParse(s, out returnValue);
+            weight = returnValue;
+
+            Console.Write("Input height (in cm): ");
+            s = Console.ReadLine();
+            float.TryParse(s, out returnValue);
+            height = returnValue;
+
+            //gender differences
+            Console.Write("F/M?: ");
+            string gender = "";
+            gender = Console.ReadLine();
+
+            if (gender == "M")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845680)